src/app/pages/schedule/schedule.ts

```diff
@@ -102,17 +112,7 @@ export class SchedulePage implements OnInit {
   confDate: string;
   showSearchbar: boolean;
 
-  constructor(
-    public alertCtrl: AlertController,
-    public confService: ConferenceService,
-    public loadingCtrl: LoadingController,
-    public modalCtrl: ModalController,
-    public router: Router,
-    public routerOutlet: IonRouterOutlet,
-    public toastCtrl: ToastController,
-    public user: UserService,
-    public config: Config
-  ) {
+  constructor() {
```

Comment:
We should be using `ngOnInit` so we can completely move away from `constructor` (since Angular prefers `inject`) but it throws a warning regarding the icons not loading in. We should look into this more since the icons did load fine so the warning is being thrown too soon.